themes/digital.gov/layouts/services/single.html

```diff
@@ -1,66 +1,57 @@
-          {{- partial "partials/core/usa-breadcrumbs.html" . -}}
-          <div class="grid-row">
-            <div class="grid-col-12">
-              {{- if .Params.kicker -}}
-                <p class="kicker">{{- .Params.kicker -}}</p>
-              {{- end -}}
-              {{/* Page Title */}}
-              <h1>{{- .Title | markdownify -}}</h1>
-
-              {{/* Deck */}}
-              {{- if .Params.summary -}}
-                <div class="deck">{{- .Params.summary | markdownify -}}</div>
-              {{- end -}}
-
-              {{- if .Params.source_url -}}
-                <p>
-                  <a
-                    href="{{- .Params.source_url -}}"
-                    title="Go to: {{- .Title | markdownify -}}"
-                    >{{- .Params.source_url -}}</a
-                  >
-                </p>
-              {{- end -}}
-            </div>
+        <div class="grid-row" data-edit-this="{{- $path -}}">
+          <div class="grid-col-12">
+            {{- if .Parent -}}
+              {{- partial "partials/core/usa-breadcrumbs.html" . -}}
+            {{- end -}}
+
+            {{- if .Params.kicker -}}
+              <p class="kicker">{{- .Params.kicker -}}</p>
+            {{- end -}}
+            {{/* Page Title */}}
+            <h1>{{- .Title | markdownify -}}</h1>
+
+            {{/* Deck */}}
+            {{- if .Params.summary -}}
+              <div class="deck">{{- .Params.summary | markdownify -}}</div>
+            {{- end -}}
+
+            {{- if .Params.source_url -}}
+              <p>
+                <a
+                  href="{{- .Params.source_url -}}"
+                  title="Go to: {{- .Title | markdownify -}}"
+                  >{{- .Params.source_url -}}</a
+                >
+              </p>
+            {{- end -}}
           </div>
         </div>
       </header>
 
-      <section>
-        <div class="grid-container grid-container-desktop">
-          <div class="grid-row tablet-lg:grid-gap-6">
-            <div class="grid-col-12 tablet:grid-col-9">
-              {{/* Content */}}
-              <div class="content">
-                {{- .Content -}}
-              </div>
-
-              {{- partial "core/page-data.html" (dict "page" . "page_type" "resource" "branch" ($.Scratch.Get "branch")) -}}
-
-            </div>
-
-            <div class="grid-col-12 tablet:grid-col-3">
-              {{/* Table of contents */}}
-              {{- partial "core/toc.html" . -}}
-
-              {{/* Last updated */}}
-              {{/* {{- partial "last-updated.html" . -}} */}}
-              {{/* Related Communities and Services */}}
-              {{- partial "core/get_related.html" . -}}
-
-              {{/* Topics */}}
-              {{/* {{- partial "core/list-topics.html" . -}} */}}
-              {{/* Share Tools */}}
-              {{/* {{- partial "core/get_sharetools.html" . -}} */}}
-            </div>
+      <div class="usa-in-page-nav-container">
```

Comment:
We should move towards a global template that has the in-page nav. This would make it easier to modify in the future.

Reply:
Created a card for this